src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html

```diff
@@ -967,25 +966,17 @@ <h3>{{ "label.final_disposition" | translate }}</h3>
       <span class="section-jj-grid-cell">
         <p>
           <span
-            style="font-size:small; color: #707070; font-weight: bold; margin: 0 !important; padding-right: 10px; float: right;">Due
-            date</span><br />
-          <ng-container *ngIf="jjDisputeInfo.status == DisputeStatus.Confirmed || jjDisputeInfo.status == DisputeStatus.Review ||

```

Comment:
So, don't we need these status checks for showing revisedDueDate anymore?

Reply:
fixed the issue